apps/windows/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "lib",
```

Comment:
the config you're extending should specify outDir already, this probably isn't necessary.

Reply:
You're right, removing in newest commit